src/App.jsx

```diff
@@ -1,36 +1,19 @@
 import { useState } from "react";
+import TodoList from "./TodoList";
+import AddTodoForm from "./AddTodoForm";
 
 // import "./App.css";
-var todoList = [
-  {
-    id: 1,
-    title: "Assignment not start",
-  },
-  {
-    id: 2,
-    title: "Assignment start",
-  },
-  {
-    id: 3,
-    title: "Assignment completed",
-  },
-];
 
 function App() {
+  const [newTodo, setNewTodo] = useState("");
+
   return (
     <>
       <div>
         <h1>Todo List</h1>
-        <ul>
-          {todoList.map(function (todolists) {
-            return (
-              <li key={todoList.id}>
-                <span>{todolists.id + " "} </span>
-                <span>{todolists.title}</span>
-              </li>
-            );
-          })}
-        </ul>
+        <AddTodoForm onAddTodo={setNewTodo} />
```

Comment:
Seems you've not defined a function to use the `setNewTodo` function to update your list of todos.

Also, it would be easier to manage if the `todoList` array was in the `App.jsx` file. That way you can easily define a method to update the list.

That's the concept of "lifting state up". This way all components that need to access that data can do so easily.


Also, remember when adding a new item to the list of todos, you'll need to come up with an `id` so that the React compiler doesn't throw an error. You can do this easily using the `Math.random` method.